public/app/plugins/datasource/elasticsearch/query_builder.js

```diff
@@ -199,29 +228,48 @@ function (queryDef) {
   };
 
   ElasticQueryBuilder.prototype.getTermsQuery = function(queryDef) {
-    var query = {
-      "size": 0,
-      "query": {
-        "filtered": {
-          "query": {
-            "query_string": {
-              "analyze_wildcard": true,
-              "query": '$lucene_query',
-            }
-          },
-          "filter": {
-            "bool": {
-              "must": [{"range": this.getRangeFilter()}]
+    var query = {};
+    if (this.esVersion >= 5) {
```

Comment:
At the time I did very very basic tests using this 5.x query construction in ES 2.4 and it seemed to work fine.
What kind of issue do you see with `analyze_wildcard = true` in 2.x ? 

I can't think of a particular case where someone needs `analyze_wildcard = true` in grafana but I also don't see any particular reason to change.